src/main/java/org/akip/domain/TemporaryProcessInstance.java

```diff
@@ -0,0 +1,133 @@
+package org.akip.domain;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+/**
+ * A TemporaryProcessInstance.
+ */
+@Entity
+@Table(name = "temporary_process_instance")
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public class TemporaryProcessInstance implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator")
+    private Long id;
+
+    @Column(name = "bpmn_process_definition_id")
+    private String bpmnProcessDefinitionId;
+
+    @Column(name = "start_date")
+    private LocalDateTime startDate;
+
+    @Column(name = "end_date")
+    private LocalDateTime endDate;
+
+    @Column(name = "status")
+    private String status;
```

Comment:
Please include a relation to the corresponding ProcessInstance (for those who ended up in a process instance)